cmd/flags.go

```diff
@@ -134,6 +134,10 @@ func addSchemaFlag(fs *pflag.FlagSet, schemaPath *string) {
 	fs.StringVarP(schemaPath, "schema", "s", "", "set schema file path")
 }
 
+func addTargetFlag(fs *pflag.FlagSet, target *util.EnumFlag) {
+	fs.VarP(target, "target", "", "set the runtime to exercise")
```

Comment:
❓ Is there a reason (a conflict with some existing flag?) not to use a shorthand? The first thing I've tried was `opa bench -t wasm <query>` and it's got to be `--target wasm`.

Reply:
There's a conflict with the `opa build` command.

Reply:
Isn't that the same flag, basically?

```
  -t, --target {rego,wasm}             set the output bundle target type (default rego)
```

Reply:
The `build` command will be a special case. I think that we should support `-t` on the other subcommands.

Reply:
The conflict is with the `opa test` command which uses `-t` for the test timeout. We could support `-t` for `bench` and `eval` if we want.

Reply:
The new `target` flag now uses `-t` as shorthand.